src/pages/release-indicators/forms/ReleaseIndicatorForm.js

```diff
@@ -0,0 +1,173 @@
+        setEditMode(true)
+        setWindowOpen(false)
+        invalidate()
+      }
+    })
+  
+    useEffect(() => {
+      ;(async function () {
+        try {
+          if (recordId) {
+            setIsLoading(true)
+  
+            const res = await getRequest({
+              extension: DocumentReleaseRepository.ReleaseIndicator.get,
+              parameters: `_recordId=${recordId}`
+            })
+            
+            setInitialData(res.record)
+          }
+        } catch (exception) {
+          setErrorMessage(error)
+        }
+        setIsLoading(false)
+      })()
+    }, [])
+
+    return (
+      <FormShell 
+          resourceId={ResourceIds.ReleaseIndicators}
+          form={formik} 
+          height={300} 
+          maxAccess={maxAccess} 
+          editMode={editMode}
+      >
+          <Grid container spacing={4}>
+              <Grid item xs={12}>
+                  <CustomTextField
+                  name='reference'
+                  label={labels.reference}
+                  readOnly = {editMode}
+                  value={formik.values.reference}
+                  required
+                  maxAccess={maxAccess}
+                  maxLength='30'
+                  onChange={formik.handleChange}
+                  onClear={() => formik.setFieldValue('reference', '')}
+                  error={formik.touched.reference && Boolean(formik.errors.reference)}
+                  helperText={formik.touched.reference && formik.errors.reference}
+                  />
+              </Grid>
+              <Grid item xs={12}>
+                    <CustomTextField
+                    name='name'
+                    label={labels.name}
+                    readOnly = {editMode}
+                    value={formik.values.name}
+                    required
+                    maxAccess={maxAccess}
+                    maxLength='30'
+                    onChange={formik.handleChange}
+                    onClear={() => formik.setFieldValue('name', '')}
+                    error={formik.touched.name && Boolean(formik.errors.name)}
+                    helperText={formik.touched.name && formik.errors.name}
+                    />
+                </Grid>
+                <Grid item xs={12}>
+                    <CustomTextField
+                    name='recordId'
+                    label={labels.id}
+                    readOnly = {editMode}
+                    value={formik.values.recordId}
+                    required
+                    maxAccess={maxAccess}
+                    maxLength='30'
+                    onChange={formik.handleChange}
+                    onClear={() => formik.setFieldValue('recordId', '')}
+                    error={formik.touched.recordId && Boolean(formik.errors.recordId)}
+                    helperText={formik.touched.recordId && formik.errors.recordId}
+                    />
+                </Grid>
+                <Grid item xs={12}>
+                <ResourceComboBox
```

Comment:
changeability is mandatory so the field should become red when saving without changeability.
Error and HelperText should check for changeability not changeabilityName.